### PR TITLE
add possiblity to localize some thing via the /lang configs

### PR DESCRIPTION
**Pull request by tomschimmel**
@TimGeyssens I've created the possiblity to localize some things via the /lang config /xml files. I utilized the localizationService which is part of Umbraco's backoffice UI. Now it's possible to localize some of the notification toaster messages and some labels.

### CODE REVIEW

**Comment by tomschimmel** [comment]
@TimGeyssens, I've resolved the merge conflict at the start of this PR. Hopefully you'll be able to check these changes. Cheers, Tom

**Comment by TimGeyssens** [comment]
looks good, thanks @fungybytes

File: src/UIOMatic/Web/UI/App_Plugins/UIOMatic/backoffice/resources/uioMaticObject.resource.js [hunk 1]
```diff
@@ -103,19 +150,19 @@
...
                     $http.post(ocBaseUrl + "Validate", angular.toJson(item)),
-                    'Failed to validate'
+					localizations.failedvalidate
                 );
 	        },
 	        getSummaryDashboardTypes: function () {
 	            return umbRequestHelper.resourcePromise(
                     $http.get(ocBaseUrl + "GetSummaryDashboardTypes"),
-                    'Failed to get summary dashboard types'
+					localization.failedsummary
```
Comment (Vertigo093i) [inline src/UIOMatic/Web/UI/App_Plugins/UIOMatic/backoffice/resources/uioMaticObject.resource.js:159]: There is a typo. Should be `localizations`. Summary dashboard is broken now.

File: src/UIOMatic/Web/UI/App_Plugins/UIOMatic/backoffice/resources/uioMaticObject.resource.js [hunk 2]
```diff
@@ -103,19 +150,19 @@
...
                 );
 	        },
 	        getTotalRecordCount: function (type) {
 	            return umbRequestHelper.resourcePromise(
-                    $http.get(ocBaseUrl + "GetTotalRecordCount?typeAlias=" + type),
-                    'Failed to get total record count'
+					$http.get(ocBaseUrl + "GetTotalRecordCount?typeAlias=" + type),
+					localization.failedtotalcount
```
Comment (Vertigo093i) [inline src/UIOMatic/Web/UI/App_Plugins/UIOMatic/backoffice/resources/uioMaticObject.resource.js:165]: The same typo as above.

**Pull request merged**